src/autoscaler/api/brokerserver/broker_handler.go

```diff
@@ -80,6 +83,27 @@ func (h *BrokerHandler) CreateServiceInstance(w http.ResponseWriter, r *http.Req
 		return
 	}
 
+	policyStr := ""
+	if body.Parameters.DefaultPolicy != nil {
+		policyStr = string(*body.Parameters.DefaultPolicy)
+	}
```

Comment:
I can't agree to Parameters.DefaultPolicy.  Referring to the bindrequestbody struct:
```
type BindingRequestBody struct {
	BrokerCommonRequestBody
	AppID  string          `json:"app_guid"`
	Policy json.RawMessage `json:"parameters,omitempty"`
}
```
we set the "parameters" directly as the policy.   But with `Parameters.DefaultPolicy` definition,  we are requesting to  the following JSON format:
```
{
 "default_policy":"xxx"
}
````
which is different from the binding experience.

Reply:
We can, I just thought that it might be useful to leave us an option to introduce more parameters if needed. But we can also remove `default_policy`.